Added instructions on how to set proper permissions on the NuGet cache folder (#4205)
umbraco/UmbracoDocs: Fundamentals/Setup/Install/iis.md

For the path you want to point it at the root of your site - where the `.csproj` file is.
:::

## Add permissions to NuGet cache folder

> You will need to change permissions for the NuGet cache folder, which is typically `C:\users\<username>\.nuget\packages`. The user or group (IIS_IUSRS) that the IIS site is running on requires Read permissions on this folder, because this is where some of the files for Umbraco and Umbraco packages are being served from during development. When the IIS user or group does not have permission to read from the NuGet cache folder you could run into a `DirectoryNotFoundException` while running the site.

Reviewer: ```suggestion
You might need to change permissions for the NuGet cache folder - `C:\users\<username>\.nuget\packages`. The user or group (IIS_IUSRS) that the IIS site is running on requires Read permissions on this folder because this is where some of the files for Umbraco and Umbraco packages are being served from during development. If the IIS user or group does not have permission to read from the NuGet cache folder, you could run into a `DirectoryNotFoundException` while running the site.
```

> When the site is published these files are copied from the NuGet cache folder to `wwwroot/umbraco` and `wwwroot/App_Plugins` and these folders will typically have the correct permissions.

Reviewer: ```suggestion
When the site is published these files are copied from the NuGet cache folder to `wwwroot/umbraco` and `wwwroot/App_Plugins` and these folders will typically have the correct permissions. For more information on setting permissions, see the [File and folder permissions](../Server-Setup/permissions.md) article.
```
